src/main/java/ch/wisv/payments/admin/products/ProductServiceImpl.java

```diff
@@ -79,9 +80,9 @@ public void addProductToGroup(Product product, ProductGroup productGroup) {
 
     @Override
     public void editProduct(ProductRequest productRequest) {
-        if (productRequest.getProductId() != 0) {
+        if (productRequest.getId() != 0) {
```

Comment:
@skamoen do you know why we have this? Why are we not allowed to edit the first product?